pkgs/development/libraries/criterion/default.nix

```diff
@@ -0,0 +1,25 @@
+{stdenv, pkgconfig, fetchFromGitHub, cmake, dyncall, nanomsg, boxfort, csptr, libgit2} :
+
+stdenv.mkDerivation {
+  name = "criterion-2.3.2";
```

Comment:
```suggestion
  pname = "criterion";
  version = "2.3.2";
```

Reply:
FYI, 2.3.3 is out now. It's not a requirement that you have it at the latest version to get this merged though.